x-pack/plugins/ml/public/shared_components/single_metric_viewer/single_metric_viewer.tsx

```diff
@@ -164,9 +152,12 @@ const SingleMetricViewerWrapper: FC<SingleMetricViewerPropsWithDeps> = ({
   );
 
   const autoZoomDuration = useMemo(() => {
-    if (!selectedJob) return;
-    return mlTimeSeriesExplorerService?.getAutoZoomDuration(selectedJob);
-  }, [mlTimeSeriesExplorerService, selectedJob]);
+    if (!selectedJobWrapper) return;
+    return mlTimeSeriesExplorerService?.getAutoZoomDuration(
+      selectedJobWrapper.job.analysis_config?.bucket_span
+    );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Unless there a rendering loop problem which this rule suppression is fixing, we should remove it. Suppressing these rules can potentially cause issues in the future.

Reply:
Agree - we should start removing these as needed as they are sprinkled throughout. Updated in 53b573ff6d8d57acfe1ca468df74f5a8d63354a3